Managed Services/getting-started-with-apache-spark.md

```diff
@@ -0,0 +1,44 @@
+{{{
+  "title": "Getting Started with Apache Spark",
+  "date": "07-28-2015",
+  "author": "Kyle Mertz",
+  "attachments": [],
+  "contentIsHTML": false
+}}}
+
+##Overview
+
+Apache Spark is available for install on a standalone CentOS 6 64-bit VM in CenturyLink Cloud.
+
+<br>
+
+***1. Navigate to the Blueprints Library and search for "spark".  Then choose the Apache Spark install Blueprint.***
```

Comment:
I felt as though it looked better and stood out more as bold/italicized because the screenshots from the blueprint below the text was bold.  If there is a requirement to stay with just bold, I can change it.